ligo-webide-frontend/base-components/packages/workspace/src/components/GistUploadModals.js

```diff
@@ -147,8 +147,10 @@ export default class GistUploadModals extends PureComponent {
       >
         {this.state.gistLink === '' && <DebouncedFormGroup
           ref={this.input}
-          label={<div>To upload your project you need to add github token</div>}
-          placeholder='Github token'
+          label={<div>To upload your project you need to add github token, or
+                      leave the default to create a gist without a Github account.
+                </div>}
+          placeholder='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'
```

Comment:
`placeholder` is just name of the field, so you need to add it to the actual value

Reply:
I changed it to `initialValue`, and I added obfuscation as well

Reply:
I changed it to `initialValue`, and I added obfuscation as well